IDEAS/Buildings/Components/InternalWall.mo

```diff
@@ -5,9 +5,10 @@ model InternalWall "interior opaque wall between two zones"
   E(y=layMul.E),
   Qgai(y=(if sim.openSystemConservationOfEnergy
          then 0 else sum(port_emb.Q_flow))),
-  final QTra_design=U_value*AWall*(TRef_a - TRef_b));
+  final QTra_design=U_value*AWall*(TRef_a - TRef_b),
+    intCon_a);
 
-  parameter Boolean linearise_b=false
+  parameter Boolean linIntCon_b=sim.linIntCon
     "= true, if convective heat transfer should be linearised"
```

Comment:
@Mathadon @damienpicard  It feels rather unnatural to set the linearisation parameters by default in the simInfomanager ...